src/coreclr/pal/src/exception/signal.cpp

```diff
@@ -471,7 +471,15 @@ static void sigfpe_handler(int code, siginfo_t *siginfo, void *context)
     invoke_previous_action(&g_previous_sigfpe, code, siginfo, context);
 }
 
-#if !HAVE_MACH_EXCEPTIONS
+#if defined(TARGET_TVOS)
+
+static bool SwitchStackAndExecuteHandler(int code, siginfo_t *siginfo, void *context, size_t sp)
```

Comment:
Why is this tvOS-specific?

Reply:
tvOS arbitrarily lacks some of the Mach APIs. I was planning to rebase and drop the tvOS part for now.